profile/README.md

```diff
@@ -1,13 +1,20 @@
-# Single sign-on is coming to the B.C. government’s GitHub organizations 
+### Welcome to the B.C. Government’s official GitHub Organization!
```

Comment:
this is only one of a few we'll have, so maybe "Welcome to the B.C. Government's open GitHub Organization"?